release.yaml

```diff
@@ -28,13 +28,10 @@ rancher-csp-adapter:
   - 2.0.3+up2.0.3-rc1
 rancher-logging:
   - 102.0.2+up3.17.10
-  - 102.0.3+up3.17.10
 rancher-logging-crd:
   - 102.0.2+up3.17.10
-  - 102.0.3+up3.17.10
 rancher-monitoring:
   - 102.0.2+up40.1.2
-  - 102.0.3+up40.1.2
 rancher-monitoring-crd:
   - 102.0.3+up40.1.2
```

Comment:
are we sure `rancher-monitoring-crd` should be at 102.0.3?, or is it supposed to be 102.0.2 after the previous revert

Reply:
github needs to add a homer "doh" emoji, or a facepalm emoji. Will be fixed shortly